.circleci/config.yml

```diff
@@ -190,6 +190,7 @@ commands:
             echo 'export FEATURE_BUSINESS_FLC=$FEATURE_BUSINESS_FLC_DEV' >> $BASH_ENV
             echo 'export FEATURE_BUSINESS_FCORP=$FEATURE_BUSINESS_FCORP_DEV' >> $BASH_ENV
             echo 'export FEATURE_BUSINESS_FNP=$FEATURE_BUSINESS_FNP_DEV' >> $BASH_ENV
+            echo 'export FEATURE_BUSINESS_NP=$FEATURE_BUSINESS_NP_DEV' >> $BASH_ENV
```

Comment:
[sand - fix this after merging]

In order for the `feature-flags` page to work in Dev, we need to set the additional env vars for staging and prod so that the dev site can read them.

This line here is fine, but we need to add the STAGING and PROD vars. For **both** NP and FNP. You can follow the pattern here on lines 195-198 and 200-203. Note how the name of the variable stays the same.